.github/workflows/setlist-parser.yml

```
@@ -0,0 +1,217 @@
+          # ベースブランチが期待するものと異なる場合、変更する
+          if [ "$current_base" != "${{ env.EXPECTED_BASE_BRANCH }}" ]; then
+            echo "Base branch is '$current_base', changing to '${{ env.EXPECTED_BASE_BRANCH }}'"
+            gh pr edit ${{ github.event.pull_request.number }} --base ${{ env.EXPECTED_BASE_BRANCH }}
+            echo "Base branch changed successfully to ${{ env.EXPECTED_BASE_BRANCH }}"
+          else
+            echo "Base branch is already correct: $current_base"
+          fi
+
+      - name: Get changed setlist files
+        id: setlist-diff
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)